src/components/LandingPage/HeroBanner/HeroBanner.component.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+    ORGANISATIONS,
+  } from '@/constants/organisations';
+  import { paths } from '@/constants/paths';
+  import Image from 'next/image';
+  import Link from 'next/link';
+
+export const HeroBanner = () => {
+    return (
+        <>
+            <div className = "bg-gradient-to-r from-verydarkblue to-darkblue rounded-[32px] min-h-[312px] text-center max-w-[1168px]">
```

Comment:
```suggestion
            <div className = "bg-gradient-to-r from-verydarkblue to-darkblue rounded-[32px] min-h-[312px] text-center w-full">
```

Nie ma powodu ustawiania fixed długości, w komponencie okalającym (rodzicu) możemy później ustalić maksymalną szerokość